### PR TITLE
Add CI runs for release/3.0

### PR DESCRIPTION
**Pull request by wtgodbe**
@safern PTAL

CC @danmosemsft @leecow

### CODE REVIEW

File: .azure-ci.yml [hunk 1]
```diff
@@ -7,6 +7,7 @@ trigger:
   branches:
     include:
     - master
+    - release/3.0
```
Comment (safern) [inline .azure-ci.yml]: Do we want release/3.0 official builds to be trigger per commit? I don't think we want that. 

cc: @mmitche

File: .azure-ci.yml [hunk 2]
```diff
@@ -7,6 +7,7 @@ trigger:
...
 
 # TODO: add paths to exclude CI when modifying docs or stuff not affecting the build
 pr:
```
Comment (safern) [inline .azure-ci.yml]: For PRs you need to add it here instead.

Comment (wtgodbe) [reply .azure-ci.yml]: Fixed

File: .azure-ci.yml (hunk 1 above)
Comment (mmitche) [reply .azure-ci.yml]: We do want that for the release branch.

Comment (safern) [reply .azure-ci.yml]: I guess the commit ratio in release branch will be pretty low?

Comment (danmosemsft) [reply .azure-ci.yml]: Very low, my guess is a few a month (a few blits and a few ports)

**Pull request merged**